### DISCUSSION TITLE
Bring Octolight to Home Assistant

### DISCUSSION THREAD

**Discussion by WorlockM** [gigibu5/OctoLight#15] [Ideas]
I don't know if this is the right place for this idea. But I would love to be able to use Octolight within the Octoprint intergration for Home Assistant, so I can create automations to turn my LEDs of or on.

**Comment by Batfinks** [comment]
It would be great to turn the pin on and off using a gcode so that we can turn it on at the beginning of a print and off when it is finished.

**Reply by gigibu5** [reply]
This could be done, but probably not soon. You can make a pull request if you would like to implement the feature.
